test/tests/props.js

```diff
@@ -82,7 +82,56 @@ describe('zoid props cases', () => {
                     }
                 })
             });
-            
+
+            return instance.render(getBody());
+        });
+    });
+
+    it('should merge props definition with input props before passing to decorator', () => {
+        return wrapPromise(({ expect }) => {
+            window.__component__ = () => {
+                return zoid.create({
+                    tag:    'test-prop-value-container-url-param-bug',
+                    url:    'mock://www.child.com/base/test/windows/child/index.htm',
+                    domain: 'mock://www.child.com',
+                    props:  {
+                        account: {
+                            type:       'string',
+                            queryParam: false,
+                            required:   true,
+                            value:      ({ props }) => {
+                                return props.account;
+                            }
+                        },
+                        testAccountParam: {
+                            type:       'string',
+                            queryParam: 'account',
```

Comment:
This query param being named `account` threw me off. Could we rename it to match the prop name? Or perhaps use `account` and `payerID` for the naming of these two props?
```suggestion
                            queryParam: 'testAccountParam',
```

Reply:
yeah absolutely. We likely don't even need to test this via a query param I just simulated the exact prop defs from messaging, however if we stick with a query param in the test will rename! Let me take a peek at the test today and how I can make it more concise/clear real quick before taking this change